ros2_generate_interface_docs/ros2_generate_interface_docs/main.py

```diff
@@ -59,6 +61,13 @@ def generate_interfaces(interfaces, html_dir, template, interface_type):
                 }
                 function_to_generate_text_from_spec = msg_utils.generate_msg_text_from_spec
 
+            if(interface_type == 'srv'):
+                documentation_data = {
+                    **documentation_data,
+                    **{'ext': 'srv', 'type': 'Service'}
+                }
```

Comment:
Why is it necessary to create a new dict here? Why not just set the `ext` and `type` in the existing dict?